### PR TITLE
2nd half of zs configs

### PR DESCRIPTION
**Pull request by Whennya**
adds configs for 57 zs maps

### CODE REVIEW

File: musicname/zs_abandoned_metro_v3.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"music"
+{
+    "madlib invazion - distant land.mp3"                    "Madlib Invazion - Distant Land"
+    "nujabes - luv sic modal soul rmx instrumental.mp3"     "Nujabes - Luv Sic (Modal Soul Rmx) Instrumental"
```
Comment (notkoen) [inline musicname/zs_abandoned_metro_v3.cfg]: I would assume `rmx` is short for remix
```suggestion
    "nujabes - luv sic modal soul rmx instrumental.mp3"     "Nujabes - Luv Sic (Modal Soul Remix) Instrumental"
```

File: musicname/zs_adrift_v4.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"music"
+{
+    "leaving_hope.mp3"                          "Nine Inch Nails - Leaving Hope"
+    "what_happened_here.mp3"                    ""
```
Comment (notkoen) [inline musicname/zs_adrift_v4.cfg]: Empty song name
```suggestion
  //"what_happened_here.mp3"                    ""
```

File: musicname/zs_aya_v2b.cfg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+"music"
+{
+    "nujabes - prayer.mp3"              "Nujabes - Prayer"
+    "haruka nakamura - lamp.mp3"        "haruka nakamura - Lamp (ft. Nujabes)"
```
Comment (notkoen) [inline musicname/zs_aya_v2b.cfg]:
```suggestion
    "haruka nakamura - lamp.mp3"        "Haruka Nakamura - Lamp (ft. Nujabes)"
```

File: musicname/zs_aya_v3a.cfg [hunk 4]
```diff
@@ -0,0 +1,6 @@
+"music"
+{
+    "nujabes - prayer.mp3"              "Nujabes - Prayer"
+    "haruka nakamura - lamp.mp3"        "haruka nakamura - Lamp (ft. Nujabes)"
```
Comment (notkoen) [inline musicname/zs_aya_v3a.cfg]:
```suggestion
    "haruka nakamura - lamp.mp3"        "Haruka Nakamura - Lamp (ft. Nujabes)"
```

File: musicname/zs_detroit_skylines_v3c.cfg
Comment (notkoen) [inline musicname/zs_detroit_skylines_v3c.cfg:1]: Can you verify if the full name of `Luv` is as what you typed? Unsure if `pt5 Instrumentals` is extra information.

File: musicname/zs_estate_v1bp.cfg
Comment (notkoen) [inline musicname/zs_estate_v1bp.cfg:1]: For this, I'm unsure if `COD: MW2` should be abbreviated as such. If you look at the `zs_oilrig` config, I wrote down `COD: Modern Warfare 2` as the name. If you want one way or the other, should enforce it for the other file then.

File: musicname/zs_festival_b5.cfg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+"music"
+{
+    "faraway_cut.mp3"           "haruka nakamura - Faraway"
```
Comment (notkoen) [inline musicname/zs_festival_b5.cfg]:
```suggestion
    "faraway_cut.mp3"           "Haruka Nakamura - Faraway"
```

File: musicname/zs_ffvii_mako_reactor_v6_2fix.cfg [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    "2w_3_ffvii_ac_jenova.mp3"                                  "FFVII: Advent Children OST - J-E-N-O-V-A"
+    "3w_1_ffvii_ac_beyond_the_wasteland.mp3"                    "FFVII: Advent Children OST - Beyond the Wasteland"
+    "3w_2_ffvii_ac_divinity_i.mp3"                              "FFVII: Advent Children OST - Divinity I"
+    "3w_3_ffvii_ac_one_winged_angel.mp3"                        "FFVII: Advent Children OST - One-Winged Angel"
+    "4w_1_pendulum_voodoo_people.mp3"                           "The Prodigy - Voodoo People (Pendulum Remix)"
+    "4w_2_pendulum_midnight_runner.mp3"                         "Pendulum - Midnight Runner"
+    "4w_3_the_qemists_stompbox.mp3"                             "The Qemists - Stompbox"
+    "4w_4_pendulum_the_other_side.mp3"                          "Pendulum - The Other Side"
+    "5w_1_pendulum_bloodsugar.mp3"                              "Pendulum - Blood Sugar"
+    "5w_2_pendulum_tarantula.mp3"                               "Pendulum - Tarantula (Pendulum vs fresh ft Spyda & Tenor Fly)"
```
Comment (notkoen) [inline musicname/zs_ffvii_mako_reactor_v6_2fix.cfg]: Not entirely sure what you should do for displaying all the information for Tarantula.
```suggestion
    "5w_2_pendulum_tarantula.mp3"                               "Pendulum - Tarantula (Pendulum vs Fresh ft. Spyda & Tenor Fly)"
```

File: musicname/zs_harvest_and_trustee_b1.cfg
Comment (notkoen) [inline musicname/zs_harvest_and_trustee_b1.cfg:1]: Don't know if you should put down `Payday 2 OST` and the actual artist, Simon Viklund.

File: musicname/zs_honkers_club_v2t.cfg
Comment (notkoen) [inline musicname/zs_honkers_club_v2t.cfg:1]: Are the two `Crockett's Theme` different? Why do I see two different artists.

File: musicname/ze_ffxiv_wanderers_palace_v6css.cfg [hunk 7]
```diff
@@ -4,7 +4,7 @@
     "ultimate_star_sky.mp3"             "Two Steps From Hell - Star Sky (Instrumental)"
     "ultimate_impossible.mp3"           "Two Steps From Hell - Impossible (ft. Merethe Soltvedt)"
     "ultimate_i_beg_you.mp3"            "Aimer - I beg you"
-    "savage_empire_of_angels.mp3"       "Thomas Bergersen - Empire of Angels (Sun)"
+    "savage_empire_of_angels.mp3"       "Two Steps from Hell - Empire of Angels (Sun)"
```
Comment (notkoen) [inline musicname/ze_ffxiv_wanderers_palace_v6css.cfg]:
```suggestion
    "savage_empire_of_angels.mp3"       "Two Steps From Hell - Empire of Angels (Sun)"
```

File: musicname/ze_sandstorm_css_v1_5x3.cfg [hunk 8]
```diff
@@ -10,7 +10,7 @@
     "level5_2.mp3"          "Transformers: Dark of the Moon OST - It's Our Fight"
     "tsfh_inv.mp3"          "Two Steps From Hell - Invincible"
     "win_l2.mp3"            "Audiomachine - Sol Invictus"
-    "win_l3.mp3"            "Thomas Bergersen - Love and Loss"
+    "win_l3.mp3"            "Two Steps from Hell - Love and Loss"
     "win_l4.mp3"            "Two Steps from Hell - For the Win"
```
Comment (notkoen) [inline musicname/ze_sandstorm_css_v1_5x3.cfg]:
```suggestion
    "win_l3.mp3"            "Two Steps From Hell - Love and Loss"
    "win_l4.mp3"            "Two Steps From Hell - For the Win"
```

File: musicname/ze_sandstorm_f1.cfg [hunk 9]
```diff
@@ -10,7 +10,7 @@
     "level5_2.mp3"          "Transformers: Dark of the Moon OST - It's Our Fight"
     "tsfh_inv.mp3"          "Two Steps From Hell - Invincible"
     "win_l2.mp3"            "Audiomachine - Sol Invictus"
-    "win_l3.mp3"            "Thomas Bergersen - Love and Loss"
+    "win_l3.mp3"            "Two Steps From Hell - Love and Loss"
     "win_l4.mp3"            "Two Steps from Hell - For the Win"
```
Comment (notkoen) [inline musicname/ze_sandstorm_f1.cfg]:
```suggestion
    "win_l4.mp3"            "Two Steps From Hell - For the Win"
```

**Review by notkoen — CHANGES_REQUESTED** [review]

**Review by notkoen — APPROVED** [review]
Something is messed up with the PR, as the files listed below are somehow deleted.
```
zs_arena_eerie_v2
zs_arena_hazardous_v3
zs_arena_obj_lms_arena_ze_hns_ttt_gm_zi2
zs_arroyo
zs_ffvii_cosmo_canyon_v3
zs_houserave
zs_lotr_minas_tirith_v4
zs_obj_insane_v9
zs_obj_npst_v14
zs_obj_npst_v7
zs_obj_rescape_v19
zs_oilrig_v1a
zs_space_alphabet_v2
zs_space_experiment_u8
zs_station_mall_v3
```

File: musicname/zs_ffvii_mako_reactor_v6_2fix.cfg (hunk 6 above)
Comment (Whennya) [reply musicname/zs_ffvii_mako_reactor_v6_2fix.cfg]: https://www.youtube.com/watch?v=NWsKQOJeL9o 
maybe should format it like this? i assume this is the original file being used 
`Pendulum & Fresh - Tarantula (ft. MC Spyda, Tenor Fly, MC Shabba)`

File: musicname/zs_detroit_skylines_v3c.cfg
Comment (Whennya) [reply musicname/zs_detroit_skylines_v3c.cfg:1]: Luv(sic) is the album iirc, i matched it directly to this: https://www.youtube.com/watch?v=BIGqhcyyBo4

File: musicname/zs_honkers_club_v2t.cfg
Comment (Whennya) [reply musicname/zs_honkers_club_v2t.cfg:1]: one of them is a remix of the Miami Vice OST i think, but i guess should add (Miami Vice OST) to the end of it

**Review by notkoen — COMMENTED** [review]
> Luv(sic) is the album iirc, i matched it directly to this: https://www.youtube.com/watch?v=BIGqhcyyBo4 

LGTM then.

**Review by notkoen — COMMENTED** [review]
> https://www.youtube.com/watch?v=NWsKQOJeL9o 
maybe should format it like this? i assume this is the original file being used 
`Pendulum & Fresh - Tarantula (ft. MC Spyda, Tenor Fly, MC Shabba)` 

What you typed is good then, yeah.

**Review by notkoen — APPROVED** [review]
LGTM

**Pull request merged**